package.nls.json

```diff
@@ -0,0 +1,205 @@
+	"githubIssues.ignoreCompletionTrigger.description": "Languages that the '#' character should not be used to trigger issue completion suggestions.",
+	"githubIssues.ignoreCompletionTrigger.items": "Language that issue completions should not trigger on '#'.",
+	"githubIssues.ignoreUserCompletionTrigger.description": "Languages that the '@' character should not be used to trigger user completion suggestions.",
+	"githubIssues.ignoreUserCompletionTrigger.items": "Language that user completions should not trigger on '@'.",
+	"githubIssues.issueBranchTitle.markdownDescription": {
+		"message": "Advanced settings for the name of the branch that is created when you start working on an issue. \n- `${user}` will be replace with the currently logged in username \n- `${issueNumber}` will be replaced with the current issue number \n- `${sanitizedIssueTitle}` will be replaced with the issue title, with all spaces and unsupported characters (https://git-scm.com/docs/git-check-ref-format) removed",
+		"comment": [
+			"{Locked=']${...}'}",
+			"Do not translate what's inside of the '${..}'. It is an internal syntax for the extension"
+		]
+	},
+	"githubIssues.useBranchForIssues.description": {
+		"message": "Determines whether a branch should be checked out when working on an issue. To configure the name of the branch, set `githubIssues.issueBranchTitle`.",
+		"comment": [
+			"{Locked=']`githubIssues.issueBranchTitle`'}",
+			"Do not translate what's inside of the `...`. It is an setting id."
+		]
+	},
+	"githubIssues.useBranchForIssues.on": "A branch will always be checked out when you start working on an issue. If the branch doesn't exist, it will be created.",
+	"githubIssues.useBranchForIssues.off": "A branch will not be created when you start working on an issue. If you have worked on an issue before and a branch was created for it, that same branch will be checked out.",
+	"githubIssues.useBranchForIssues.prompt": "A prompt will show for setting the name of the branch that will be created and checked out.",
+	"githubIssues.issueCompletionFormatScm.markdownDescription": {
+		"message": "Sets the format of issue completions in the SCM inputbox. \n- `${user}` will be replace with the currently logged in username \n- `${issueNumber}` will be replaced with the current issue number \n- `${issueNumberLabel}` will be replaced with a label formatted as #number or owner/repository#number, depending on whether the issue is in the current repository",
+		"comment": [
+			"Do not translate what's inside of the ${...}. It is an internal syntax for the extension."
+		]
+	},
+	"githubIssues.workingIssueFormatScm.markdownDescription": {
+		"message": "Sets the format of the commit message that is set in the SCM inputbox when you **Start Working on an Issue**. Defaults to `${issueTitle} \nFixes #${issueNumber}`",
+		"comment": [
+			"Do not translate what's inside of the ${...}. It is an internal syntax for the extension."

```

Comment:
this won't be picked up by pseudo language since it's auto-generated. Your call if that matters or not.

Reply:
Seems fine so long as it gets picked up for real translation.